C/CUDA/CUDA_full@10.2/build_tarballs.jl

```diff
@@ -18,6 +18,19 @@ sources_windows = [
                "b538271c4d9ffce1a8520bf992d9bd23854f0f29cee67f48c6139e4cf301e253", "installer.exe")
 ]
 
+version_linux_aarch64 = v"10.2.300"
+sources_linux_aarch64 = [
+    AptSource(
+        "https://repo.download.nvidia.com/jetson/common",
+        "r32.6",
+        ["main"],
+        "http://repo.download.nvidia.com/jetson/jetson-ota-public.asc",
```

Comment:
Are you actually using this key for anything?

Reply:
Not yet - the intent is to check the signature of the repo's Release and Package files at some point.

The whole AptSource is a bit of a hack at this point in time - WIP.

Reply:
Removed (parked at https://github.com/IHPSystems/Yggdrasil/tree/stemann/cuda_10.2_aarch64-linux-gnu_with_apt_pkg_source )